PowerControl/Controller.cs

```diff
@@ -299,6 +307,35 @@ private Task NeptuneDevice_OnInputReceived(hidapi.HidDeviceInputReceivedEventArg
             return new Task(() => { });
         }
 
+        private void PowerControlLoopTimer_Tick(object sender, EventArgs e)
+        {
+            if (powerControlLoopTimer is null)
+                return;
+
+            try
+            {
+                powerControlLoopTimer.Enabled = false;
+
+                //if (TDP.Instance != null
```

Comment:
System.AccessViolationException: 'Attempted to read or write protected memory. This is often an indication that other memory is corrupt.' if I uncomment this. Not familiar with memory management so fixing may take some time.

Reply:
Resolved. There's no need in storing the separate "current TDP" value. I can simply switch to 10W TDP in PowerControl if I detect the Silent fan profile being selected (which detection is already implemented in PowerControl and is working fine).